web/vim_lib.js

```diff
@@ -657,7 +657,29 @@ var LibraryVIM = {
     };
     /* VIMJS_FOLD_END */
 
+    vimjs.lastMouseDownTarget = vimjs.canvas_node; // set focus on start
+
+    var ignoreKeys = function() {
+      var retval = (vimjs.lastMouseDownTarget !== vimjs.canvas_node);
```

Comment:
Ok. I shortened it now. I was looking at some similar code where there could be a problem with undefined checks, that's why it was like that.